.github/workflows/build-node.yml

```diff
@@ -0,0 +1,43 @@
+# To use, configure github repo settings -> Secrets:
+#   * DOCKER_REGISTRY   eg. docker.io:port/ (optional, leave unset for dockerhub, trailing slash REQUIRED)
+#   * DOCKER_ORGANIZATION
+#   * DOCKER_USERNAME
+#   * DOCKER_PASSWORD
+#   * GIT_REPO
+
+
+name: Build Docker Node
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  job0:
+    name: Build Docker Node
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Determine Meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ secrets.DOCKER_REGISTRY }}${{ secrets.DOCKER_ORGANIZATION }}/node
+          tags: |
+            type=sha,format=long,prefix=
+            type=ref,event=tag
+            type=ref,event=branch
+            type=ref,event=pr
```

Comment:
Cannot follow here. Could you expand with a comment block?

Nothing ever sets `latest`. That should probably be set for end user convenience upon a release (tagged non-preview version).